### PR TITLE
Volume rendering with colormaps

### PR DESCRIPTION
**Pull request by izaid**

### CODE REVIEW

**Comment by Eric89GXL** [comment]
Is this good to go from your end (other than the PEP8 error at the end of a file)?

**Comment by izaid** [comment]
Yep! This, and my emulated 3d texture PR, can be merged. I just rebased them off master.

File: vispy/visuals/volume.py [hunk 1]
```diff
@@ -698,4 +711,4 @@ def draw(self, transforms):
             self._program['u_threshold'] = self._threshold
         
         # Draw!
-        self._program.draw('triangles', self._index_buffer)
+        self._program.draw('triangles', self._index_buffer)
```
Comment (Eric89GXL) [inline vispy/visuals/volume.py]: Travis will be mad that this doesn't have a trailing newline, consider using `make flake` to check for other probs

File: examples/basics/scene/volume.py [hunk 2]
```diff
@@ -66,6 +72,10 @@ def on_key_press(event):
     elif event.text == '3':
         volume1.visible = not volume1.visible
         volume2.visible = not volume1.visible
+    elif event.text == '4':
+        cmap = cmaps.next()
```
Comment (Eric89GXL) [inline examples/basics/scene/volume.py]: I get an error that it has no attribute `next` -- this is a Python3 thing. You should use `from vispy.ext.six import next` and do `next(cmaps)`

**Comment by almarklein** [comment]
Also it fails on the galaxy/galaxy.py test. This one is fixed in master, so you probably need a rebase.

**Comment by almarklein** [comment]
Other than that. LGTM, this is awesome!

**Pull request merged**